hyperlane-mailbox/tests/harness.rs

```diff
@@ -180,3 +210,51 @@ async fn test_latest_checkpoint() {
     // The index is 0-indexed
     assert_eq!(index, 0u32);
 }
+
+#[tokio::test]
+async fn test_process() {
+    let (mailbox, _id, recipient) = get_contract_instance().await;
+
+    let metadata = vec![5u8; 100];
+    let message_body = vec![6u8; 100];
+
+    let agent_message = test_message(&mailbox, recipient);
+
+    let process_call = mailbox
+        .methods()
+        .process(
+            metadata,
+            ContractMessage {
+                version: agent_message.version,
+                nonce: agent_message.nonce,
+                origin: agent_message.origin,
+                sender: h256_to_bits256(agent_message.sender),
+                destination: TEST_ORIGIN_DOMAIN,
+                recipient: Bits256(*recipient),
+                body: message_body
+            }
+        )
+        .call()
+        .await;
+
+    let res = match process_call {
+        Ok(_response) => String::from("success"),
+        Err(error) => get_revert_string(error)
+    };
+    println!("REVERT REASON: {}", res);
```

Comment:
@tkporter we are getting "!version" here

Reply:
See https://github.com/hyperlane-xyz/fuel-contracts/pull/33